client.js

```diff
@@ -45,101 +41,335 @@ bot.on('message', function (message) {
     // listen for messages that will start with `!`
     if (message.content.startsWith('!')) {
         console.log(message.content);
-        let colIndex = message.content.indexOf(':');
-        let cmd;
+        const spaceIndex = message.content.indexOf(' ');
+        let cmd = '';
         let args;
-        if (colIndex === -1) {
-            cmd = message.content.substring(1).toUpperCase().trim();
-        } else {
-            cmd = message.content.substring(1, colIndex).toUpperCase();
-            args = message.content.substring(colIndex+1).toUpperCase().split(',');
+        //allow users to enter single word commands with no spaces
+        if (spaceIndex === -1) {
+            cmd = message.content.substring(1).toUpperCase();
+        } else  {
+            cmd = message.content.substring(1, spaceIndex).toUpperCase();
+            args = message.content.substring(spaceIndex+1).toUpperCase().split(',');
             args.forEach((elem, index, array) => array[index] = elem.trim().replace(/[<>"\{\}\[\]]/g, ''));
         }
 
+        //if user entered a depreciated command, inform them
+        const deprecatedCmds = ['ADD', 'END', 'SHOW'];
+        let abort = false;
+        deprecatedCmds.forEach((deprecatedCmd) => {
+            if (deprecatedCmd === cmd) {
+                message.author.send(`It looks like you've entered a deprecated command.  Please see the current commands below:${helpMsg}`)
+                abort = true;
+            }
+        })
+
+        if (abort) return;
+
+        //if user entered a current command ending in ':', inform them and trim the colon off
+        const colonCmds = ['WATCH:', 'UNWATCH:', 'BLOCK:', 'UNBLOCK:', 'SNOOZE:', 'UNSNOOZE:', 'LIST:', 'WATCHES:', 'EXTEND:', 'HELP:'];
```

Comment:
I found myself still using colons a lot, so I wanted the command to still execute but inform the user.